lib/cache/cache.js

```diff
@@ -314,7 +314,7 @@ class Cache {
     let innerRequest = null
 
     // 2.
-    if (typeof request !== 'string') {
+    if (request instanceof Request) {
```

Comment:
Is this fetch request?

Reply:
yes it is, I import it so it's not the global one